docker/Dockerfile

```diff
@@ -26,24 +26,6 @@ COPY . $GOPATH/src/github.com/go-debos/debos
 WORKDIR $GOPATH/src/github.com/go-debos/debos/cmd/debos
 RUN go install ./...
 
-# Install the latest archlinux-keyring, since the one in Debian is bound
-# to get outdated sooner or later.
-# WARNING: returning to the debian package will break the pacstrap action
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-        pkgconf \
-        python3-all \
-        sq \
-        systemd \
-        make && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN git clone https://gitlab.archlinux.org/archlinux/archlinux-keyring && \
```

Comment:
the problem is not so much arch; the problem is trying to rebuild their keyring. The debian packaging for this simply uses the prebuild keyrings instead (part of every release)